packages/@react-aria/tabs/src/TabsKeyboardDelegate.ts

```diff
@@ -21,45 +21,31 @@ export class TabsKeyboardDelegate<T> implements KeyboardDelegate {
 
   constructor(collection: Collection<T>, direction: Direction, orientation: Orientation, disabledKeys: Set<Key> = new Set()) {
     this.collection = collection;
-    this.flipDirection = direction === 'rtl' && orientation === 'horizontal';
```

Comment:
> @LFDanLu The orientation variable is no longer used in the code, unsure of plans for future use. Keeping it unused causes Typescript issues, which can be suppressed. What do you think?

I think we'll want to keep `this.flipDirection = direction === 'rtl' && orientation === 'horizontal';` here to keep ArrowLeft/Right behavior in vertical Tabs in LTR/RTL locales in line with similar components (ActionGroup).

Reply:
Understood, I'll update it to match the behaviors of similar components. However, in RTL left is next for horizontal orientation, but the semantics flip when in vertical orientation, where left then means previous. This seems confusing to the end user. Would you suggest I open an RFC for this?

Reply:
Updated to match similar component behaviors. 🙂

Reply:
> Understood, I'll update it to match the behaviors of similar components. However, in RTL left is next for horizontal orientation, but the semantics flip when in vertical orientation, where left then means previous. This seems confusing to the end user. Would you suggest I open an RFC for this?

Good point, this makes sense to me and got me revisiting the logic. Looks like we actually had this working [here](https://github.com/adobe/react-spectrum/pull/223) where ArrowLeft/ArrowUp always moved to the previous item and ArrowRight/ArrowDown always moved to the next item in RTL (can be observed [here](https://reactspectrum.blob.core.windows.net/reactspectrum/046831ed00bf013c6f728bf474b270bf5656b208/storybook/index.html?path=/story/buttongroup--default)). However, looks like the behavior changed in [this PR](https://github.com/adobe/react-spectrum/pull/339) to be what it currently is, presumably as a fix for [something separate](https://github.com/adobe/react-spectrum/pull/339/commits/c2c46911ce92cd478b0945492029d93869c905b8). This feels like the wrong behavior as you pointed out, would you mind opening a separate issue for this? 

For this PR, we'll keep it as is to avoid scope creep and so we can discuss with the rest of the team if the current behavior is incorrect.